Makefile

```diff
@@ -53,7 +53,10 @@ test-unit-cover: ARGS=-timeout=10m -tags='$(UNIT_TEST_TAGS)' -coverprofile=$(TES
 test-race: ARGS=-timeout=10m -race -tags='$(TEST_RACE_TAGS)'
 $(TEST_TARGETS): run-tests
 
-run-tests:
+copy-env:
+	@cp .env.example .env.test
```

Comment:
I meant to have a static, commited `.env.test` rather than copying an example one.

Reply:
Done